src/components/Repository/FileList.js

```diff
@@ -146,6 +147,44 @@ class FileList extends Component {
         document.body.removeChild(a);
     }
 
+    cleanResults = () => {
+        // Create copy so that tableData is not modified

```

Comment:
We haven't talked about this much...but we typically do not include a lot of comments. in the code.
In this case, I would remove all the comments but the "CSV output discards null columns..." because that may be unexpected behavior and it explains why the code is there, rather than the what of the code.